### PR TITLE
[WIP] CIRCLE-14082 update replicated and docker doc

### PR DESCRIPTION
**Pull request by mqarty**
# Description
Publish the documentation from https://paper.dropbox.com/doc/Updating-server-replicated-version--AS3rjV5tdfVIpUpnWG6LjhxKAg-wiB1Ir2awxAD41zQMc7t7.

# Reasons
https://circleci.atlassian.net/browse/CIRCLE-14082

### CODE REVIEW

**Comment by polymeris** [comment]
The content LGTM, the formatting probably needs a bit of work (not sure how the doc's MD parser works, exactly), e.g. surround example outputs with triple backticks.

File: jekyll/_cci2/updating-server-replicated-version.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+
+# Updating server replicated version
+
+## Assumptions
+- Your installation is Ubuntu 14.04-based
+- You are running replicated version 2.10.3 on your services machine
+  - replicated --version
+- Your installation is **not** airgapped and you can access the internet from it
+- All steps take place on the services machine
+## Preparations
```
Comment (polymeris) [inline jekyll/_cci2/updating-server-replicated-version.md]: Might need newline before header (some markdown parsers do)

File: jekyll/_cci2/updating-server-replicated-version.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+
+    replicatedctl app stop
+
+Application shutdown takes a few minutes. Please check the administration dashboard, and wait for the status to become “Stopped” before continuing. You can also run
+
+    replicatedctl app status inspect
+
+to check the current status of the application.
+Example Output:
+[
```
Comment (polymeris) [inline jekyll/_cci2/updating-server-replicated-version.md]: Should be surrounded with triple backticks to format as code ` ``` `

File: jekyll/_cci2/updating-server-replicated-version.md [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+Restart app with
+
+    replicatedctl app start
+
+The application will take a few minutes to spin up. You can check the progress in the administration dashboard or by executing;
+
+    replicatedctl app status inspect
+
+Example output:
+[
```
Comment (polymeris) [inline jekyll/_cci2/updating-server-replicated-version.md]: ditto

**Comment by mqarty** [comment]
@polymeris , made changes based on your comments. Please take a look.

**Pull request merged**